noir/noir-repo/acvm-repo/acvm_js/src/execute.rs

```diff
@@ -165,7 +166,10 @@ async fn execute_program_with_native_type_return(
     foreign_call_executor: &ForeignCallHandler,
 ) -> Result<WitnessStack, Error> {
     let program: Program = Program::deserialize_program(&program)
-    .map_err(|_| JsExecutionError::new("Failed to deserialize circuit. This is likely due to differing serialization formats between ACVM_JS and your compiler".to_string(), None))?;
+    .map_err(|_| JsExecutionError::new(
+        "Failed to deserialize circuit. This is likely due to differing serialization formats between ACVM_JS and your compiler".to_string(), 
+        None ,
```

Comment:
```suggestion
        None,
```
looks like we can run fmt on this, I would expect this to be caught

Reply:
Not sure why but `cargo fmt` and `cargo fmt --all` is not formatting this file :/